mslice/workspace/workspace.py

```diff
@@ -0,0 +1,77 @@
+import numpy as np
+import operator
+from mantid.simpleapi import CreateWorkspace
+
+
+class Workspace(object):
+
+    def __init__(self, matrix_workspace):
+        self.inner_workspace = matrix_workspace
+
+    def get_coordinates(self):
```

Comment:
Can we have some method documentation here and on the other methods, please? See https://www.python.org/dev/peps/pep-0257/